datafusion/core/tests/sqllogictests/test_files/window.slt

```diff
@@ -2204,3 +2204,78 @@ SELECT SUM(c12) OVER(ORDER BY c1, c2 GROUPS BETWEEN 1 PRECEDING AND 1 FOLLOWING)
 2.994840293343 NULL
 9.674390599321 NULL
 7.728066219895 NULL
+
+# test_c9_rn_ordering_alias
```

Comment:
Maybe worth a comment here saying  the intention is there is no `SortExec` after the `BoundedWindowAggExec`